akka-http-core/src/main/scala/akka/http/scaladsl/settings/ParserSettings.scala

```diff
@@ -129,6 +133,21 @@ object ParserSettings extends SettingsCompanion[ParserSettings] {
       }
   }
 
+  trait IllegalResponseHeaderValueProcessingMode extends akka.http.javadsl.settings.ParserSettings.IllegalResponseHeaderValueProcessingMode
```

Comment:
Let's make it `sealed`. We're not able to support other ones than configured ones anyway